uint32 index
itm_nonlinear_mpc/itm_trajectory_point[] traj
---
bool success